lambdas/models/config.py

```diff
@@ -7,3 +7,7 @@ def to_camel(string: str) -> str:
 
 
 conf = ConfigDict(alias_generator=to_camel)
+
+
+def to_capwords(snake_case_string: str) -> str:
+    return "".join(x.capitalize() for x in snake_case_string.split("_"))
```

Comment:
I installed a package called inflection for a previous ticket which may come in use here rather than creating our own transformation functions on strings [](https://inflection.readthedocs.io/en/latest/) can use the camelize function from the library and then capitalize? Up to you though just thought you might be interested in taking a look at what it can do :)

Reply:
Thanks!  It worked nicely as a drop in replacement :D   A great util library to know
I have updated this part to use inflection